frontend/src/features/public-form/components/FormFields/FormFields.tsx

```diff
@@ -258,7 +262,11 @@ export const FormFields = ({
           )}
         <PublicFormPaymentResumeModal />
         {/* TODO: (respondent copy): Remove when respondent copy is out of beta */}
-        {form?.hasRespondentCopy && isRespondentCopyEnabled ? (
+        {form &&
+        ((form?.hasRespondentCopy && isRespondentCopyEnabled) ||
+          ('hasStatusTracker' in form &&
+            form?.hasStatusTracker &&
+            isStatusTrackerCopyEnabled)) ? (
```

Comment:
Feels like something along the lines of this would be sufficient. 
```suggestion
        {
        (form?.hasRespondentCopy && isRespondentCopyEnabled) ||
            (form?.hasStatusTracker && isStatusTrackerCopyEnabled)
           ) ? (
```